app/app/components/navigation/BreadCrumbs.tsx

```diff
@@ -0,0 +1,34 @@
+// define the Props
+export type CrumbItem = {
+  title: string;
+  href: string;
+};
+
+export type BreadcrumbsProps = {
+  items: CrumbItem[];
+};
+
+const Breadcrumbs = ({ items }: BreadcrumbsProps) => {
+  return (
+    <>
+      <div className="flex gap-2 items-start">
+        {items.map((crumb, i) => {
```

Comment:
```suggestion
        {items && items.map((crumb, i) => {
```
I'm getting "something went wrong" when I log out and go to the home page. It seems to be resolved when I make this change. I'm guessing it's not liking when there are no items to show.

A quick skim (as time allowed) and everything else seems to be good. Nice work getting this in so quick! And 👍 to Tailwind and Headless too.